tsconfig.json

```diff
@@ -1,15 +1,15 @@
 {
-   "compilerOptions": {
-      "lib": [
-         "es5",
-         "es6"
-      ],
-      "target": "es5",
-      "module": "commonjs",
-      "moduleResolution": "node",
-      "outDir": "./build",
-      "emitDecoratorMetadata": true,
-      "experimentalDecorators": true,
-      "sourceMap": true
-   }
-}
\ No newline at end of file
+	"compilerOptions": {
+		"experimentalDecorators": true,
```

Comment:
Did you really had to reformat all files? It makes harder to understand your changes.
How about to keep the original formatting?